crates/biome_analyze/src/lib.rs

```diff
@@ -448,19 +447,15 @@ where
                     let index =
                         self.suppressions
                             .line_suppressions
-                            .binary_search_by(|suppression| {
-                                if suppression.text_range.end() < entry.text_range.start() {
-                                    Ordering::Less
-                                } else if entry.text_range.end() < suppression.text_range.start() {
-                                    Ordering::Greater
-                                } else {
-                                    Ordering::Equal
-                                }
+                            .partition_point(|suppression| {
+                                suppression.text_range.end() < entry.text_range.start()
```

Comment:
The comment that starts at line 430 is now outdated and must be updated to describe the new logic

Reply:
ok, i will update this~

Reply:
done~

Reply:
`biome_js_analyze` shouldn't be here because we didn't add any fix there. The fix was only in the `biome_analyze` crate. We just added a test in the js crate, but it could have been in any language crate

Reply:
remove it~

Reply:
> done~

It doesn't seem the comment is updated